### ISSUE TITLE
find out why logging-from-nodejs recipe is causing gh-pages failure

### ISSUE THREAD

**Issue by bear** [logstash/cookbook#72]
Can someone post here the error messages (if any) that are being generated so I can help research this gh-pages issue

thanks

**Comment by jordansissel** [comment]
what uses gh pages in this project? cookbook.logstash.net is not hosted on github pages anymore (for sanity reasons, weird things kept failling which would take the whole site offline on commits)

**Comment by bear** [comment]
oh, apologies if that was a bad assumption!

I thought it was from the conversation on IRC when it was first added and for some reason thought that.

**Comment by jordansissel** [comment]
Yeah I originally used gh-pages for this (and logstash.net) but after some frustrating (and confusing) failures I gave up. Easier to run it myself so at least I know why it's failling ;)

(for clarity, gh-pages says it fails to build on every commit, but it works fine when I deploy it on cookbook.logstash.net)

**Comment by electrical** [comment]
Ahhh okay. then will just ignore it. sorry for all the extra commits for fixing something that doesn't need to be fixed.
